pkg/vsphere/tasks/waiter.go

```diff
@@ -164,11 +201,11 @@ func logFault(op trace.Operation, fault types.BaseMethodFault) {
 }
 
 func logSoapFault(op trace.Operation, fault types.AnyType) {
-	op.Debugf("unexpected soap fault on task retry : %#v", fault)
+	op.Debugf("unexpected soap fault on task retry : %s", fault)
```

Comment:
somehow, print out all fields can help us understand what's wrong. Why change it back to %s, and curious why not for the above logFault?